Integrations/EDL/README.md

```diff
@@ -0,0 +1,58 @@
+This integration provides External Dynamic List (EDL) as a service for the system indicators (Outbound feed).
+
+
+## Use Cases
+---
+1. Generateg feeds to be used on PAN-OS as External Dynamic Lists.
+2. Create External Dynamic Lists (EDLs) to track on AutoFocus the IP addresses, URLs and domains used by ransomware, known APT groups, and active malware campaigns.
+3. Create External Dynamic Lists to track the IPs and URLs used by Microsoft Office365, or used as tor exit nodes, or used by CDNs and cloud services.
+
+## Configure EDL on Demisto
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for EDL.
```

Comment:
Isn't the `display` name for this integration something else now? Something with `PAN-OS` in it?

Reply:
Yeah, Palo Alto Networks PAN-OS EDL Service. you could change it to that, though EDL should also do